### ISSUE TITLE
[Improvement] Submit the module to packagist.org

### ISSUE THREAD

**Issue by lamasfoker** [dadolun95/magento2-sib-core#2]
To improve the accessibility of your module you can submit it at [packagist.org](https://packagist.org/packages/submit). In this way the user does not have to add a repository node to his composer.json.

**Comment by dadolun95** [comment]
Hi @lamasfoker,
I added core, contact-sync and order-sync modules to packagist.org
I also updated the README of the modules adding a preferential route for installation via composer with repman.io
